kvrocks.conf

```diff
@@ -462,6 +462,18 @@ profiling-sample-record-threshold-ms 100
 # 0-7am every day.
 compaction-checker-range 0-7
 
+# When the compaction checker is triggered, the db will periodically pick the SST file
+# with the highest "deleted percentage" (i.e. the percentage of deleted keys in the SST 
+# file) to compact, in order to free disk space. 
+# However, if a specific SST file was created before "force-compact-file-age" seconds ago, 
```

Comment:
I would replace `was created before "force-compact-file-age" seconds ago` with `was created more than "force-compact-file-age" seconds ago`.

Reply:
Thanks, made the change.